core/src/main/java/psiprobe/tools/AccessorFactory.java

```diff
@@ -43,7 +43,13 @@ public static Accessor getInstance() {
   private static Accessor getReflective() {
     try {
       return new ReflectiveAccessor();
-    } catch (Exception ex) {
+    } catch (ClassNotFoundException e) {
```

Comment:
I've got this already...I was waiting on on Pull request which @MALfunction84 just merged.  I'll review what I have and submit full logging coverage next.  I've even switched everything to use {} on loggers to prevent early processing in case logging is turned off.